ClimbingGearBackend/Migrations/ClimbingGearContextModelSnapshot.cs

```diff
@@ -70,8 +70,8 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<int>("Quantity")
                         .HasColumnType("integer");
 
-                    b.Property<long>("UserId")
-                        .HasColumnType("bigint");
+                    b.Property<string>("UserId")
+                        .HasColumnType("text");
```

Comment:
Why is this text? Isn't the UserId a number?

Reply:
No, Identity uses UUIDs for their user IDs. So they are all strings.